src/db/models/basket.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+from sqlalchemy import Column, DateTime, Integer, JSON, ForeignKey
+from sqlalchemy.orm import Relationship
+from db.conection import Base
+
+class Bascet(Base):
+    __tablename__ = "baskets"
+    
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, ForeignKey('users.id'), nullable=False)
+    smartphone_id = Column(Integer, ForeignKey('smartphones.id'), nullable=False)
+    smartphones = Column(JSON, nullable=False)
```

Comment:
If we create a basket_smartphone_association, so why do we need smartphones column?

Reply:
removed